docs/scripting/trust_scripts/users.top.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: users.top
+---
+
+> Displays top 10 users by GC.
```

Comment:
```suggestion
Displays top 10 users by GC.
```

Reply:
The templates now include the carrot bracket styling to match the upgrade template/styling. The previously merged pages will need to be edited to match.